### ISSUE TITLE
Crash in numbers test

### ISSUE THREAD

**Issue by timholy** [JuliaDebug/JuliaInterpreter.jl#71]
```julia
julia> @interpret 2646693125139304345//842468587426513207 < pi
Unreachable reached at 0x7f58cd99668e

signal (4): Illegal instruction
in expression starting at no file:0
top-level scope at ./none:0
jl_fptr_trampoline at /usr/local/julia/julia-1.1/src/gf.c:1864
jl_toplevel_eval_flex at /usr/local/julia/julia-1.1/src/toplevel.c:758
jl_toplevel_eval_in at /usr/local/julia/julia-1.1/src/toplevel.c:793
eval at ./boot.jl:328 [inlined]
evaluate_foreigncall! at /home/tim/.julia/dev/JuliaInterpreter/src/interpret.jl:161
...
```

**Comment by tkoolen** [comment]
Reduced to `@interpret Base.decompose(big(pi))`, which calls `memcpy`.

**Comment by timholy** [comment]
Thanks! #72

**Comment by timholy** [comment]
Closed by #72
